### PR TITLE
Add myCoolPythonProgram.

### PR DESCRIPTION
**Pull request by kuraudoh**
Hey guys please check out my code.

### CODE REVIEW

**Comment by weebwater** [comment]
It's aiight.

**Comment by utemtilsin** [comment]
Woah, what a cool program.

File: myCoolPythonProgram.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("this is so cool.")
```
Comment (kuraudoh) [inline myCoolPythonProgram.py:1]: Oh my god this is a masterpiece.

Comment (oddsbodkin) [reply myCoolPythonProgram.py:1]: This is fucking horseshit

Comment (NealNAL) [reply myCoolPythonProgram.py:1]: this is fucking horseshit

Comment (utemtilsin) [reply myCoolPythonProgram.py:1]: You solved the internet.

**Review by kuraudoh — COMMENTED** [review]
Hey guys review this.

**Pull request closed without merging**